### ISSUE TITLE
[BUG] Node key cannot be composed from properties of differing types

### ISSUE THREAD

**Issue by Cole-Greer** [nodestream-proj/nodestream#274]
**Describe the bug**
When attempting to compose a node key from properties of differing types, nodestream will attempt to sort all key values which will throw an exception if the value types are not comparable.

**To Reproduce**
1. Create an interpretation which creates a node key with different datatypes, for example:
```
    interpretations:
      - type: source_node
        node_type: Person
        key:
          name: John
          age: 23
```
2. When running the pipeline, an error such as this will be raised:
```
TypeError: '<' not supported between instances of 'int' and 'str'
```
The source of this error is https://github.com/nodestream-proj/nodestream/blob/7254d8e109270065e609b1da97aa1db2bcfffa61/nodestream/model/graph_objects.py#L115

**Expected behavior**
Nodestream should be able to construct a key of mixed types.

**Comment by zprobst** [comment]
@Cole-Greer thanks for the catch... we'll take a look at this one.

**Comment by zprobst** [comment]
This is covered by #275 which will be released with `0.12`
